packages/plugins/integration-plane-ui/src/lib/components/plane-settings/plane-settings.component.html

```
@@ -0,0 +1,143 @@
+						</button>
+					</div>
+				}
+			</form>
+
+			@if (!isEditing()) {
+				<div class="actions-section col-xl-6 col-12">
+					<hr />
+					<div class="d-flex align-items-center justify-content-between">
+						<span class="label">{{ 'INTEGRATIONS.PLANE_PAGE.REGENERATE_KEY_LABEL' | translate }}</span>
+						<button
+							nbButton
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)